src/univariate/continuous/arcsine.jl

```diff
@@ -23,18 +23,19 @@ External links
 
 * [Arcsine distribution on Wikipedia](http://en.wikipedia.org/wiki/Arcsine_distribution)
 
+Use `Arcsine(a, b, ::NoArgCheck)` to bypass argument checks.
 """
 struct Arcsine{T<:Real} <: ContinuousUnivariateDistribution
     a::T
     b::T
-
-    Arcsine{T}(a::T, b::T) where {T} = (@check_args(Arcsine, a < b); new{T}(a, b))
 end
 
-Arcsine(a::T, b::T) where {T<:Real} = Arcsine{T}(a, b)
+Arcsine(a::T, b::T) where {T} = (@check_args(Arcsine, a < b); new{T}(a, b))
```

Comment:
`new` is not valid in constructors defined outside of the type definition block.